content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -231,13 +185,42 @@ Events:
   13s          13s         1        {kubelet 172.17.4.201}    spec.initContainers{init-myservice}     Normal        Created       Created container with docker id 5ced34a04634; Security:[seccomp=unconfined]
   13s          13s         1        {kubelet 172.17.4.201}    spec.initContainers{init-myservice}     Normal        Started       Started container with docker id 5ced34a04634
 ```
+
+To see logs for the init containers in this Pod, run:
 ```shell
 kubectl logs myapp-pod -c init-myservice # Inspect the first init container
 kubectl logs myapp-pod -c init-mydb      # Inspect the second init container
 ```
 
-Once we start the `mydb` and `myservice` services, we can see the Init Containers
-complete and the `myapp-pod` is created:
+At this point, those init containers will be waiting to discover Services named
+`mydb` and `myservices`.
+
+This Service configuration would create those:
```

Comment:
"This ... would create those" ...

Can we revise this sentence?

Reply:
Revised